internal/server/evaluation/evaluation.go

```diff
@@ -91,3 +112,56 @@ func (s *Server) Boolean(ctx context.Context, r *rpcevaluation.EvaluationRequest
 
 	return resp, nil
 }
+
+// Batch takes ina list of *evaluation.EvaluationRequest and returns their respective responses.
+func (s *Server) Batch(ctx context.Context, b *rpcevaluation.BatchEvaluationRequest) (*rpcevaluation.BatchEvaluationResponse, error) {
+	resp := &rpcevaluation.BatchEvaluationResponse{
+		Responses: []*rpcevaluation.EvaluationResponse{},
+	}
+
+	for _, req := range b.GetRequests() {
+		f, err := s.store.GetFlag(ctx, req.NamespaceKey, req.FlagKey)
+		if err != nil {
+			var errnf errs.ErrNotFound
+			if b.GetExcludeNotFound() && errors.As(err, &errnf) {
+				continue
+			}
```

Comment:
I think we need to put our heads together on this logic a bit.

This used to work for old batch because we put all the details of the call in the response.
However, we chose to elide that information this time around, mostly because we forgot about this particular usecase.

So, if you do hit a not found with this enabled, you will get a smaller response set that the request set and you wont know which responses are for which requests.
This was fine before because the response payload had all the request details.
I assume callers would then do the work to match them back up on the other side with the original requests.

If we want to move away from the style of the old evaluator (for this new endpoint), we could make it so there is always at-least one entry in the batch for each request and that they're aligned. When we want to step over not found we could preserve the not found reason this particular case.

Options moving forward:
- Go back to the old RPC style where every response has its request data in it.
- Reconsider what goes into the batch.

It is a new API, so we can approach this how we want.
To be honest, the old way seems harder to handle than having an alligned response set with the request set.
Having to correlated requested flag key, entity ID and arbitrary context maps to match request with response must be a pain.

cc @markphelps